python-backend/requirements.txt

```diff
@@ -29,3 +29,4 @@ sqlalchemy-filters==0.8.0
 uwsgi==2.0.17.1
 uwsgitop==0.10
 Werkzeug==0.14.1
+PyJWT==1.7.1
```

Comment:
Is this used, I don't see it imported anywhere.

Reply:
It's imported as `jwt`